inz-personal-data/Issues/birthdate/description.xml

```diff
@@ -0,0 +1,38 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo INZ Personal Data Birthdate</en>
+        <nl>Demo INZ Personal Data Birthdate</nl>
```

Comment:
I would recommend merging this credential with the age limits: they both describe the same thing, just with different granularity.

Reply:
Thanks for the suggestions. But we are intentionally using them separately.